widgets/nuxeo-date-picker.html

```diff
@@ -175,24 +187,26 @@
       },
 
       _pickerValueChanged: function() {
-        if (this._pickerValue === undefined) {
-          return;
-        }
-        if (this._wasOpened && this._pickerValue && moment(this._pickerValue).isValid()) {
-          this.set('value', moment(this._pickerValue).toJSON());
-          this._wasOpened = false;
+        if (this._pickerValue !== undefined && !this._preventPickerUpdate) {
+          var date = moment(this._pickerValue);
+          if (this._wasOpened && this._pickerValue && date.isValid()) {
+            this.set('value', date.toJSON());
+            this._wasOpened = false;
+          }
         }
+        this._preventPickerUpdate = false;
       },
 
       _inputValueChanged: function() {
-        if (this._inputValue === undefined) {
-          return;
-        }
-        if (this._inputValue && moment(this._inputValue).isValid()) {
-          this.set('value', moment(this._inputValue).toJSON());
-        } else {
-          this.set('value', '');
+        if (this._inputValue !== undefined && !this._preventInputUpdate) {
+          var date = moment(this._inputValue);
+          if (this._inputValue && date.isValid()) {
```

Comment:
you already validate if `this._inputValue` exists in the first if so you shouldn't need to check here